iree/compiler/Translation/LLVM/test/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+iree_lit_test_suite(
+  NAME
+    lit
+  SRCS
+    "expr.mlir"
+  DATA
+    iree::tools::IreeFileCheck
+    iree::tools::iree-opt
+    iree::tools::iree-translate
```

Comment:
Should this be `mlir-translate` instead?

Reply:
IIUC we use iree/tools for lit tests here not mlir

Reply:
My bad, sorry. Seems I added the comment to the wrong line...